definitions/variable/energy/tag_scope.yaml

```diff
@@ -0,0 +1,8 @@
+- scope:
+    - process:
+        description: the scope of process emissions (within Scope 1)
+    - indirect:
+        description: the scope of indirect emissions (i.e., Scope 2 emissions)
+    - total:
```

Comment:
The convention in variable names is to not add "Total" as an identifier but have it implicit, like "Emissions|CO2" is the sum (total) of all emissions.